docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md

```diff
@@ -9,24 +9,74 @@ modeled in [Decision Model and Notation](https://www.omg.org/dmn/) (DMN).
 
 ![task](assets/business-rule-task.png)
 
-Business rule tasks behave exactly like [service tasks](/components/modeler/bpmn/service-tasks/service-tasks.md). Both
-task types are based on jobs and [job workers](/components/concepts/job-workers.md).
+:::note
+Camunda Cloud supports alternative task implementations for the business rule task. If you want to
+use your own implementation for a business rule task, see the **Alternative task implementation**
+section below. The sections before it apply to the DMN decision implementation only.
+:::
+
+When the process execution arrives at a business rule task, a decision is evaluated using the
```

Comment:
:wrench: We should use the term `process instance` instead. In the other docs, we don't use the term `process execution`. We should use the same terms to make it easier for the users.

Reply:
It's not common, but it is still used in the User Task and Manual Task documentation. Shall I take those along?

Reply:
:sweat_smile: We don't need to change the other places immediately. In the long run, I would prefer consistent naming. 
I guess this is something that we could document and communicate better.

Reply:
I'd prefer to change them now. Often new documentation is written using the existing documentation as a baseline and example. If we don't want to use this wording, let's remove it.